src/Data/Auth/KeycloakProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Bullseye, Spinner } from "@patternfly/react-core";
+import { ReactKeycloakProvider } from "@react-keycloak/web";
+import Keycloak, { KeycloakConfig } from "keycloak-js";
+
+export interface AuthConfig extends KeycloakConfig {
```

Comment:
Why is the AuthConfig extending KeycloakConfig instead of being generic?

Reply:
What we receive from the config.js file is the KeyclokaConfig extended by one extra property. Should I rename it?

Reply:
yes, rename it please